ratpack-bearer-auth/src/main/java/st/ratpack/auth/springsec/SpringSecCheckTokenValidator.java

```diff
@@ -93,4 +107,16 @@ private String buildBasicAuthHeader(String user, String password) {
 		return "Basic " + encodedCreds;
 	}
 
+	private static ObjectMapper buildObjectMapper() {
+		ObjectMapper objectMapper = new ObjectMapper();
+		objectMapper.registerModule(new Jdk7Module());
+		objectMapper.registerModule(new Jdk8Module());
+		objectMapper.registerModule(new GuavaModule());
+		objectMapper.setPropertyNamingStrategy(PropertyNamingStrategy.CAMEL_CASE_TO_LOWER_CASE_WITH_UNDERSCORES);
+		objectMapper.getFactory().enable(JsonParser.Feature.ALLOW_UNQUOTED_FIELD_NAMES);
+		objectMapper.getFactory().enable(JsonParser.Feature.ALLOW_SINGLE_QUOTES);
+		objectMapper.disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
+		return objectMapper;
+	}
```

Comment:
This makes the ObjectMapper no longer configurable via Guice injection.  

I did this to address the specific nature of the JSON being served up by spring w/ snake cased properties.
Other possible options:
a) leave it configurable and utilize ```@JsonProperty``` annotations.
b) Don't use ```@JsonProperty``` and instead require ObjectMapper to be configured as expected.